code/game/gamemodes/cult/runes.dm

```diff
@@ -1,1053 +1,1005 @@
-			break
-		if(R.word1==wordblood && R.word2==wordsee && R.word3==worddestr) //conceal
-			T = new(src.loc)
-			T.imbue = "conceal"
-			imbued_from = R
-			break
-		if(R.word1==wordhell && R.word2==worddestr && R.word3==wordother) //armor
-			T = new(src.loc)
-			T.imbue = "armor"
-			imbued_from = R
-			break
-		if(R.word1==wordblood && R.word2==wordsee && R.word3==wordhide) //reveal
-			T = new(src.loc)
-			T.imbue = "revealrunes"
-			imbued_from = R
-			break
-		if(R.word1==wordhide && R.word2==wordother && R.word3==wordsee) //deafen
-			T = new(src.loc)
-			T.imbue = "deafen"
-			imbued_from = R
-			break
-		if(R.word1==worddestr && R.word2==wordsee && R.word3==wordother) //blind
-			T = new(src.loc)
-			T.imbue = "blind"
-			imbued_from = R
-			break
-		if(R.word1==wordself && R.word2==wordother && R.word3==wordtech) //communicat
-			T = new(src.loc)
-			T.imbue = "communicate"
-			imbued_from = R
-			break
-		if(R.word1==wordjoin && R.word2==wordhide && R.word3==wordtech) //communicat
-			T = new(src.loc)
-			T.imbue = "runestun"
-			imbued_from = R
-			break
-	if (imbued_from)
-		for (var/mob/V in viewers(src))
-			V.show_message("<span class='danger'>The runes turn into dust, which then forms into an arcane image on the paper.</span>", 3)
-		usr.say("H'drak v[pick("'","`")]loso, mir'kanas verbot!")
-		qdel(imbued_from)
-		qdel(newtalisman)
+	//Begin revival
+	if(!in_range(mob_to_sacrifice,src))
+		user << "<span class='warning'>The sacrificial target has been moved!</span>"
+		fail_invoke()
+		log_game("Raise Dead rune failed - catalyst corpse moved")
+		return
+	if(!(mob_to_revive in T.contents))
+		user << "<span class='warning'>The corpse to revived has been moved!</span>"
+		fail_invoke()
+		log_game("Raise Dead rune failed - revival target moved")
+		return
+	if(mob_to_sacrifice.stat != DEAD)
+		user << "<span class='warning'>The sacrifical target must be dead!</span>"
+		fail_invoke()
+		log_game("Raise Dead rune failed - catalyst corpse is not dead")
+		return
+	if(user.name == "Herbert West")
+		user.say("To life, to life, I bring them!")
 	else
-		return fizzle()
+		user.say("Pasnar val'keriam usinar. Savrae ines amutan. Yam'toth remium il'tarat!")
+	mob_to_sacrifice.visible_message("<span class='warning'><b>[mob_to_sacrifice] disintegrates into black mist. The mist lingers, and then flows into [mob_to_revive]!</span>")
+	mob_to_sacrifice.dust()
+	sleep(20)
+	if(mob_to_revive.stat != DEAD || !mob_to_revive)
+		visible_message("<span class='warning'>The black mist rises and dissipates.</span>")
+	mob_to_revive << "<span class='cult'>\"PASNAR SAVRAE YAM'TOTH. Arise.\"</span>"
+	mob_to_revive.visible_message("<span class='warning'>[mob_to_revive] draws in a huge breath, red light shining from their eyes.</span>", \
+								  "<span class='userdanger'>You awaken suddenly from the void. You're alive!</span>")
+	mob_to_revive.revive() //This does remove disabilities and such, but the rune might actually see some use because of it!

```

Comment:
The potential runtime is still here, null.stat , null.visible_message() , null.revive()